cog/comment.py

```
@@ -20,7 +20,7 @@
 
 try:
     with open(
-        f"{os.getcwd()}/DataBase/server.config.json", "r", encoding="utf-8"
+        f"{os.getcwd()}/database/server.config.json", "r", encoding="utf-8"
```

Ruff (0.14.7)

Unnecessary mode argument

Remove mode argument

(UP015)